src/javascript/app_2/App/Components/Elements/Notifications/__tests__/notifications.spec.js

```diff
@@ -0,0 +1,22 @@
+import React                  from 'react';
+import { expect }             from 'chai';
+import { configure, shallow } from 'enzyme';
+import Adapter                from 'enzyme-adapter-react-16';
+import { Notifications }      from '../notifications.jsx';
+
+configure({ adapter: new Adapter() });
+
+describe('Notifications', () => {
+    it('should render one <Notifications /> component', () => {
+        const wrapper = shallow(<Notifications />);
+        expect(wrapper).to.have.length(1);
+    });
+    it('should render .no-notifications-container when the list is not passed', () => {
+        const wrapper = shallow(<Notifications />);
+        expect(wrapper.find('.no-notifications-container').exists()).to.be.true;
+    });
+    it('should not render .no-notifications-container when the list is passed', () => {
+        const wrapper = shallow(<Notifications list={['First', 'Second', 'Third']} />);
```

Comment:
I think one of the pros of writing tests is to find out code smells and improve the code as a result. This is one of those cases, I am not sure if that would fall under this very same PR or not, but while we are writing tests, it is better to refactor as well.

For example, here, instead of looking for 'attached classes' we can check if the component contains `<Notification />` or `<EmptyNotification />`. So we can refactor the code and change the following: 

```jsx
<div className='no-notifications-container'>
    <div className='notification-message'>
        <div className='bell-icon'>
            <IconBell />
        </div>
        <div>
             <h4>{localize('No Notifications')}</h4>
             <span className='no-notifications-message'>{localize('You have yet to receive any notifications')}</span>
        </div>
    </div>
</div>
```

to Something like this: 
```jsx
<EmptyNotification />
```

And then Test the `<EmptyNotification />` component by itself. 

> Note to self: Writing tests at this stage (when the development is considered as done) has always comes with dangerous areas. On of those areas that we should be all aware of while writing them is when we just try to pass the tests and adjust them with the current code, while we should be targeting the correct behaviour. If tests are failing and we get red (and we know that test is right). That is more than alright to just modify the code and not the test itself.